### PR TITLE
Java: Add `Zrevrank` command.

### PR DESCRIPTION
**Pull request by SanHalacogluImproving**

### CODE REVIEW

File: java/integTest/src/test/java/glide/standalone/TransactionTests.java [hunk 1]
```diff
@@ -121,4 +123,21 @@ public void lastsave() {
         var response = client.exec(new Transaction().lastsave()).get();
         assertTrue(Instant.ofEpochSecond((long) response[0]).isAfter(yesterday));
     }
+
+    @Test
+    @SneakyThrows
+    public void zrank_zrevrank_withscores() {
+        String zSetKey1 = "{key}:zsetKey1-" + UUID.randomUUID();
+        Transaction transaction = new Transaction();
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("7.2.0")) {
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/standalone/TransactionTests.java:132]:
```suggestion
        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("7.2.0"));
```
And move it to the top

File: java/integTest/src/test/java/glide/standalone/TransactionTests.java [hunk 2]
```diff
@@ -121,4 +123,21 @@ public void lastsave() {
...
+            transaction.zadd(zSetKey1, Map.of("one", 1.0, "two", 2.0, "three", 3.0));
+            transaction.zrankWithScore(zSetKey1, "one");
+            transaction.zrevrankWithScore(zSetKey1, "one");
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/standalone/TransactionTests.java:135]: nit
```suggestion
            Transaction transaction = new Transaction()
                    .zadd(zSetKey1, Map.of("one", 1.0, "two", 2.0, "three", 3.0))
                    .zrankWithScore(zSetKey1, "one")
                    .zrevrankWithScore(zSetKey1, "one");
```

File: java/integTest/src/test/java/glide/standalone/TransactionTests.java [hunk 3]
```diff
@@ -121,4 +123,21 @@ public void lastsave() {
         var response = client.exec(new Transaction().lastsave()).get();
         assertTrue(Instant.ofEpochSecond((long) response[0]).isAfter(yesterday));
     }
+
+    @Test
+    @SneakyThrows
+    public void zrank_zrevrank_withscores() {
```
Comment (Yury-Fridlyand) [inline java/integTest/src/test/java/glide/standalone/TransactionTests.java:129]: repeat this test for cluster client

**Review by Yury-Fridlyand — APPROVED** [review]

File: java/integTest/src/test/java/glide/standalone/TransactionTests.java (hunk 2 above)
Comment (SanHalacogluImproving) [reply java/integTest/src/test/java/glide/standalone/TransactionTests.java:135]: This seems to be the convention so far are we changing it?

Comment (Yury-Fridlyand) [reply java/integTest/src/test/java/glide/standalone/TransactionTests.java:135]: nope, both variants are good

**Review by Yury-Fridlyand — APPROVED** [review]
LGTM
Please remind me why we can't test all functions in our common transaction test in TransactionTestUtilities.java?

**Pull request merged**